configs/config.yaml

```
@@ -0,0 +1,22 @@
+train:
+  batch_size: 128
+  epochs: 10
+  log_interval: 10
+  lr: 0.001
+  loss_function: "MSE"
+
+model:
+  type: "AE"
+  embedding_size: 32
+
+logger:
+  results_path: "results/"
+  checkpoint_path: "checkpoints/"
+  enable_wandb: False
+
+system:
+  no_cuda: False
```

```suggestion
  cuda: True
```

Double negatives are a source of confusion for the human mind :)